### PR TITLE
Fix for issues with 'cordova run' command.

### LINKED ISSUE #87: 'cordova run' command issue
# Bug Report
When running `cordova run` with a declared platform, everything is good.
However, on `cordova run` without any platform declaration, cordova will try to run all platforms.
Let's say that browser runs first and android runs second. When it's the turn of android to run, it will search for target `default` which was set in browser platform arguments. Since both platforms will use these arguments, android will never complete its build as it never finds `default` target that was intended for `cordova-browser` only.
### What is expected to happen?
Run all platforms without issues.

### What does actually happen?
Android platform crashes.

### Command or Code
Run `cordova run` command without any platform declaration.
Problem is located in https://github.com/apache/cordova-browser/blob/master/bin/template/cordova/lib/run.js.

### Version information
Cordova 9.0.0

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

### PR DESCRIPTION
**Pull request by DimitrisRK**
Do not change values of args since it's used from the rest of platforms during 'cordova run' command execution which runs all platforms.

### Motivation and Context
#87 

### Description
fixes #87

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Comment by breautek** [comment]
To link this PR as a fix for another issue, you must use one of the keywords defined [here](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords).

Can you please update your PR description so that it uses one of these keywords.

E.g.

fixes <issue_link>

**Comment by DimitrisRK** [comment]
Done! Thanks a lot for pointing out my mistake!

File: bin/template/cordova/lib/run.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ var cordovaServe = require('cordova-serve');
 
 module.exports.run = function (args) {
```
Comment (erisu) [inline bin/template/cordova/lib/run.js]:
```suggestion
module.exports.run = function ({port = 8000, target = 'default', silent : noLogOutput = false} = {}) {
```

File: bin/template/cordova/lib/run.js [hunk 2]
```diff
@@ -26,9 +26,9 @@ var cordovaServe = require('cordova-serve');
...
     // defaults
-    args.port = args.port || 8000;
-    args.target = args.target || 'default'; // make default the system browser
-    args.noLogOutput = args.silent || false;
+    var port = args.port || 8000;
+    var target = args.target || 'default'; // make default the system browser
+    var noLogOutput = args.silent || false;
```
Comment (erisu) [inline bin/template/cordova/lib/run.js]: These can be deleted if you include the changes above. This also includes the comment and new line.

File: bin/template/cordova/lib/run.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ module.exports.run = function (args) {
     }
 
     var server = cordovaServe();
-    server.servePlatform('browser', { port: args.port, noServerInfo: true, noLogOutput: args.noLogOutput })
+    server.servePlatform('browser', { port: port, noServerInfo: true, noLogOutput: noLogOutput })
```
Comment (erisu) [inline bin/template/cordova/lib/run.js]:
```suggestion
    server.servePlatform('browser', { port, noServerInfo: true, noLogOutput })
```

File: bin/template/cordova/lib/run.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ module.exports.run = function (args) {
 
             console.log('startPage = ' + startPage);
             console.log('Static file server running @ ' + projectUrl + '\nCTRL + C to shut down');
-            return server.launchBrowser({ 'target': args.target, 'url': projectUrl });
+            return server.launchBrowser({ 'target': target, 'url': projectUrl });
```
Comment (erisu) [inline bin/template/cordova/lib/run.js]:
```suggestion
            return server.launchBrowser({ target, url: projectUrl });
```

**Pull request closed without merging**